src/pkg/backup/backup_test.go

```diff
@@ -26,6 +28,84 @@ func TestBackupUnitSuite(t *testing.T) {
 	suite.Run(t, &BackupUnitSuite{Suite: tester.NewUnitSuite(t)})
 }
 
+func (suite *BackupUnitSuite) TestBackup_Type() {
+	table := []struct {
+		name string
+		// versionStart is the first version to run this test on. Together with
+		// versionEnd this allows more compact test definition when the behavior is
+		// the same for multiple versions.
+		versionStart int
+		// versionEnd is the final version to run this test on. Together with
+		// versionStart this allows more compact test definition when the behavior
+		// is the same for multiple versions.
+		versionEnd int
+		// Take a map so that we can have check not having the tag at all vs. having
+		// an empty value.
+		inputTags map[string]string
+		expect    string
+	}{
+		{
+			name:         "NoVersion Returns Empty Type If Untagged",
+			versionStart: version.NoBackup,
+			versionEnd:   version.NoBackup,
+		},
+		{
+			name:         "PreTag Versions Returns Merge Type If Untagged",
+			versionStart: 0,
+			versionEnd:   version.All8MigrateUserPNToID,
+			expect:       model.MergeBackup,
+		},
+		{
+			name:         "Tag Versions Returns Merge Type If Tagged",
+			versionStart: version.All8MigrateUserPNToID,
+			versionEnd:   version.Backup,
+			inputTags: map[string]string{
+				model.BackupTypeTag: model.MergeBackup,
+			},
+			expect: model.MergeBackup,
+		},
+		{
+			name:         "Tag Versions Returns Assist Type If Tagged",
+			versionStart: version.All8MigrateUserPNToID,
+			versionEnd:   version.Backup,
+			inputTags: map[string]string{
+				model.BackupTypeTag: model.MergeBackup,
+			},
+			expect: model.MergeBackup,
```

Comment:
This test case should be against `model.AssistBackup` right?

Reply:
yes! Thanks for the catch on my copy/paste error